scripts/templates/profiles/tooltip.ejs

```diff
@@ -0,0 +1,7 @@
+<div class="js-infowindow-title title"><%= title %></div>
```

Comment:
Can you move this file to templates/shared please? I'm going to use it for the Tool as well.